src/components/Main.js

```diff
@@ -11,27 +11,26 @@ import { fetchEvents } from '../actions/eventActions';
 
 
 class Main extends Component {
-  componentWillMount(){
-
-    this.props.dispatch(fetchEvents());
-  }
-
-  render() {
-    console.log("From the main file ",this.props.events);
-    return (
-      <div>
-        <Navigation />
-        <EventList events ={this.props.events}/>
-        <Footer />
-      </div>
-    )
-  }
+	componentWillMount(){
```

Comment:
Seeing the docs here, https://reactjs.org/docs/react-component.html#componentdidmount, what do you think of the debate on where to instantiate our network requests?